src/components/form/form.js

```diff
@@ -522,10 +540,14 @@ class Form extends React.Component {
     }
 
     if (valid && this.props.onSubmit) {
-      this.props.onSubmit(ev);
+      this.props.onSubmit(ev, this.enableForm);
```

Comment:
We should also pass this to `afterFormValidation` above ^

Reply:
It's just an idea but shouldn't we just depricate afterFormValidation?

Reply:
We may want to do something/show something to the user extra when the client side validation fails.
`if valid` check might be useful although maybe that is a rare case?

Reply:
Yeah its just a hook that gives them more control if they need it.

Reply:
Will passing `this.enableForm` as a callback function work? Or will the `this` inside `enableForm` be pointing elsewhere when it gets called?

Reply:
should be ok as `enableForm` is a class property, so should be automatically bound to the instance of the component

Reply:
I just booted up the code I wrote to test it in GAC. It works fine.